acceptance/framework/vault/helpers.go

```diff
@@ -83,6 +83,7 @@ func GenerateGossipSecret() (string, error) {
 // ConfigureGossipVaultSecret generates a gossip encryption key,
 // stores it in Vault as a secret and configures a policy to access it.
 func ConfigureGossipVaultSecret(t *testing.T, vaultClient *vapi.Client) string {
+	vaultClient.SetNamespace("admin")
```

Comment:
I am glad you asked!   That is a remnant from when I first started setting up this test. The namespace gets set [here](https://github.com/hashicorp/consul-k8s/pull/1171/files#diff-1bf5d9747e4b81163df612b5ea14549234f67e8484a9dee10ff85cce2bc64786R138-R141).  After it is set on the Vault client, any commands from the vault client after this will get run in that namespace.